presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -107,6 +161,44 @@ public static double ceiling(@SqlType(StandardTypes.DOUBLE) double num)
         return Math.ceil(num);
     }
 
+    private static SqlScalarFunction decimalCeilingFunction(String name)
```

Comment:
this is more complicated as we discussed offline, will do it in a subsequent pull request / commit, lets ignore this for now.